abs-docs/asciidoc/oo.adoc

```diff
@@ -74,9 +74,10 @@ block has the same restrictions as in the init block.
 .Syntax
 |====
 | _ClassDecl_ ::= | `class` _SimpleTypeIdentifier_ [ `(` [ _ClassParameterList_ ] `)` ] [ `implements` _InterfaceList_ ]
-|                 | `{` [ _FieldDeclList_ ] [ _Block_ ] [_RecoveryBlock_] [ _MethDeclList_ ] `}`
+|                 | `{` [ _TraitUseList_ ] [ _FieldDeclList_ ] [ _Block_ ] [_RecoveryBlock_] [ _MethDeclList_ ] `}`
 | _ClassParameterList_ ::= | _Type_ _SimpleIdentifier_ { `,` _Type_ _SimpleIdentifier_ }
 | _InterfaceList_ ::= | _TypeIdentifier_ { `,` _TypeIdentifier_ }
+| _TraitUseList_ ::= | adds _TraitName_`;` { `,` adds _TraitName_ `;`}
```

Comment:
Need a space between _ and ` in _TraitName`;`
Also, the comma `,` is not in the grammar as defined in ABS.g4